usagov/vote-gov-drupal
# Feature/VOTE-3074 tome increase path count patch

## Jira ticket

[VOTE-3074](https://cm-jira.usa.gov/browse/VOTE-3074)

## Description

Apply patch to mitigate issues resulting from tome static generation using page-count > 1.

## Deployment and testing

### Post-deploy steps

n/a

### QA/Testing instructions

1. Upon generating the static site review site to confirm there are no visual issues.
2. Review sidebar navigation for interior pages to confirm they are complete and functional. (this was the initial issue that caused us to set page-count=1)

## Checklist for the Developer




## Checklist for the Peer Reviewers
